docs/samples/shards/UI/Table/1.edn

```diff
@@ -0,0 +1,37 @@
+(defloop main-wire
+  (GFX.MainWindow
+   :Contents
+   (->
+    (Setup
+     (GFX.DrawQueue) >= .ui-draw-queue
+     (GFX.UIPass .ui-draw-queue) >> .render-steps)
+    .ui-draw-queue (GFX.ClearQueue)
+
+    (UI
+     .ui-draw-queue
+     (UI.CentralPanel
+      :Contents
+      (->
+       [{:Name "Doe" :Surname "John"}
+        {:Name "Dough" :Surname "Jane"}
+        {:Name "Smith" :Surname "Dick"}]
+       (UI.Table
+        :Resizable true
+        :Striped true
+        :RowIndex .index
+        :Columns
+        [{:Initial (float 20)}
```

Comment:
Maybe the Row callback can even be part of this table, keeps them tied closer together
like 
```clj
    {:Header "Surname" :Value (-> ...)}
```

Reply:
I was thinking about it but I find it easier to read when it's separate.

Reply:
It's also easier to compose.